### PR TITLE
(GH-75) Add support for multiple repository definition

### LINKED ISSUE #75: Enhance policy definition with repositories
It would be nice if we can define [repositories ](https://github.com/bbtsoftware/AzureDevOpsPolicyConfigurator/blob/1f3897b0b4db7f910106dc5c2812454625e2a423/src/AzureDevOpsPolicyConfigurator/Data/Policy.cs#L92)for a policy, not only one repository.
Same way as we have branch and branches.

### PR DESCRIPTION
**Pull request by Alea81**
Fixes #75

### CODE REVIEW

**Comment by Alea81** [comment]
Fixes #75

File: src/AzureDevOpsPolicyConfigurator/Helpers/PolicyExtension.cs [hunk 1]
```diff
@@ -112,6 +111,37 @@ public static IEnumerable<Policy> FlattenBranches(this IEnumerable<Policy> polic
...
+        /// <param name="policies">Policies</param>
+        /// <param name="serializer">Json serializer</param>
+        /// <returns>Flattened policies</returns>
+        public static IEnumerable<Policy> FlattenRepositories(this IEnumerable<Policy> policies, IJsonSerializer serializer)
+        {
+            var newPolicies = new List<Policy>();
+
+            foreach (var policy in policies)
+            {
+                if (policy.Repositories != null && policy.Repositories.Count > 0)
```
Comment (x-jokay) [inline src/AzureDevOpsPolicyConfigurator/Helpers/PolicyExtension.cs]: `policy.Repositories.Count > 0` can be removed.

File: src/AzureDevOpsPolicyConfigurator.Tests/MultiRepositoryDefinitionTests.cs [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                    {
+                        "myrepo1", "myrepo2"
+                    }
+                }
+            };
+
+            policies = policies.FlattenRepositories(serializer).ToList();
+
+            Assert.Equal(2, policies.Count);
+            Assert.True(policies[0].Repository == "myrepo1");
```
Comment (x-jokay) [inline src/AzureDevOpsPolicyConfigurator.Tests/MultiRepositoryDefinitionTests.cs]: Use `Assert.Equal()` for comparison.

```cs
Assert.Equal("myrepo1", policies[0].Repository);
```

File: src/AzureDevOpsPolicyConfigurator.Tests/MultiRepositoryDefinitionTests.cs [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            Assert.Null(policies[0].Repositories);
+            Assert.True(policies[1].Repository == "myrepo2");
```
Comment (x-jokay) [inline src/AzureDevOpsPolicyConfigurator.Tests/MultiRepositoryDefinitionTests.cs]: Use `Assert.Equal()` for comparison.

```cs
Assert.Equal("myrepo2", policies[1].Repository);
```

**Pull request merged**